MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -34,15 +34,17 @@ namespace mu2e {
     MCRelationship _rel; // relationship of this particle to its primary
     uint16_t _nhits; // number of associated StrawHits
     uint16_t _nactive; // number of associated active hits
-    XYZVectorF _mom; // initial momentum
+    XYZTVectorF   _mom; // initial momentum
+    CLHEP::Hep3Vector _pos; // initial position
```

Comment:
Is this Hep3Vecror still in the schema?  If so, please change to XYZVectorF.

Reply:
no I just forgot to push that file, its changed